drivers/SmartThings/zigbee-switch/src/robb/init.lua

```diff
@@ -0,0 +1,71 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+local capabilities = require "st.capabilities"
+
+local constants = require "st.zigbee.constants"
+
+local SimpleMetering = zcl_clusters.SimpleMetering
+local ElectricalMeasurement = zcl_clusters.ElectricalMeasurement
+
+local ROBB_DIMMER_FINGERPRINTS = {
+  { mfr = "ROBB smarrt", model = "ROB_200-011-0" },
+  { mfr = "ROBB smarrt", model = "ROB_200-014-0" }
+}
+
+local function is_robb_dimmer(opts, driver, device)
+  for _, fingerprint in ipairs(ROBB_DIMMER_FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+      return true
+    end
+  end
+  return false
+end
+
+local function energy_meter_handler(driver, device, value, zb_rx)
+  local raw_value = value.value
+  local multiplier = device:get_field(constants.SIMPLE_METERING_MULTIPLIER_KEY) or 1
+  local divisor = device:get_field(constants.SIMPLE_METERING_DIVISOR_KEY) or 1000000
```

Comment:
you could just change your `added` method to set this field to 1000000 and avoid repeating the defaults here

Reply:
simplified, cause it's not really needed.